apps/examples/src/examples/CustomConfigExample/ui-overrides.ts

```diff
@@ -8,7 +8,7 @@ export const uiOverrides: TLUiOverrides = {
 		tools.card = {
 			id: 'card',
 			icon: 'color',
-			label: 'Card' as any,
```

Comment:
Labels can now be a string, removing the `TLTranslationKey` type as it's not extensible.